### PR TITLE
Update docs on config file location

### PR DESCRIPTION
**Pull request by uuf6429**
Addendum to https://github.com/Behat/Behat/pull/1464.

@stof I'm not used to RST format, or even the code style...so please let me know if I missed anything.

### CODE REVIEW

**Review by acoulton — APPROVED** [review]
Thanks! I'm not really familiar with .rst either, but the diff looks plausible and the text is clear. Unless someone more experienced with the docs wants to take a look I vote we merge it and see how it looks 🤣

**Comment by carlos-granados** [comment]
@uuf6429 @acoulton

I opened the file in an online rst viewer and it did not look correct, see the image. 
Screenshot 2024-10-24 at 09 01 15
But maybe whatever generates the HTML files for the site interprets this differently. It would be good to know how this HTML is exactly generated and to learn how to do it locally for testing. @everzet @stof do you have any info that you could share about this process? Thanks!

**Comment by uuf6429** [comment]
@carlos-granados good call! Using [documatt](https://snippets.documatt.com/), I got the same result as you did.

After adding some blank lines, it seemed to render well (so I've pushed the changes).

**Review by acoulton — APPROVED** [review]

**Comment by acoulton** [comment]
Thanks for checking it properly @carlos-granados - viewing the updated file in documatt it looks good - I've also discovered that on desktop clicking "View file" from the pull request diff shows the whole file rendered by github which also looks good.

**Review by carlos-granados — APPROVED** [review]
Looking good now, thanks @uuf6429

File: user_guide/configuration.rst [hunk 1]
```diff
@@ -13,8 +13,16 @@ profiles.
...
-format. Behat tries to load ``behat.yml`` or ``config/behat.yml`` by default,
-or you can tell Behat where your config file is with the ``--config`` option:
+format. By default, Behat loads the configuration from the first file matching:
+
+* ``behat.yaml`` or ``behat.yml``
+* ``behat.yaml.dist`` or ``behat.yml.dist``
+* ``behat.dist.yaml`` or ``behat.dist.yml``
+* ``config/behat.yaml`` or ``config/behat.yml``
+* ``config/behat.yaml.dist`` or ``config/behat.yml.dist``
+* ``config/behat.dist.yaml`` or ``config/behat.dist.yml``
```
Comment (stof) [inline user_guide/configuration.rst:23]: should this be an ordered list instead of an unordered list ?

Comment (uuf6429) [reply user_guide/configuration.rst:23]: I mean.... now that you're saying it, it obviously makes sense. 😆

Comment (uuf6429) [reply user_guide/configuration.rst:23]: Done: https://github.com/Behat/docs/pull/166

**Pull request merged**